mw-webapp/src/logic/wayPage/reportsTable/WayColumns.module.scss

```diff
@@ -38,11 +38,11 @@ $listLeftMargin: 20px;
   margin-bottom: $defaultMargin;
 }
 
-.comments li {
+.comments > li {
```

Comment:
could we use specific class name for lists?

Reply:
maybe